packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-file-upload/question.js

```diff
@@ -166,27 +171,24 @@ class MultiFileUploadQuestion extends Question {
 		}
 	};
 
-	async #saveFilesToBlobStorage(req, journeyResponse) {
+	async #saveFilesToBlobStorage(files, journeyResponse) {
 		let result = [];
 
-		if (this.fieldName in req.files) {
-			// const validFiles = getValidFiles(errors, body.files[this.fieldName]);
-			// eslint-disable-next-line no-restricted-syntax
-			for await (const file of req.files[this.fieldName]) {
-				const document = await createDocument(
-					{
-						id: journeyResponse.journeyId,
-						referenceNumber: journeyResponse.referenceId
-					},
-					file,
-					file.name,
-					this.documentType.name
-				);
+		// eslint-disable-next-line no-restricted-syntax
+		for await (const file of files) {
```

Comment:
I'm not sure we need to do for await because we're awaiting the result in the for loop.

Reply:
have replaced for await with for